Sources/TripKit/model/TKBookingTypes.swift

```diff
@@ -326,6 +343,13 @@ extension TKBooking {
 
     /// Maximum number of tickets that can be purchased of this fare
     public var max: Int?
+    
+    /// list of riders under the fare for filtering purposes
+    @DefaultEmptyArray public var riders: [TKBooking.Rider]
+    
+    public let status: Status?
+    
+    public let type: RideType
```

Comment:
Can you please make this optional, as this change is breaking the tests?

Reply:
will do